run/testing/events_pubsub.e2e_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package testing
```

Comment:
Might want a more specific package name than this -- at least different from the standard library `testing` package.

Reply:
Switching to `cloudruntests`